android/src/fabric/java/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -46,6 +46,9 @@ public NativeProxy(ReactApplicationContext context) {
                         fabricUIManager);
         prepareLayoutAnimations(LayoutAnimations);
         installJSIBindings();
+        if(BuildConfig.DEBUG){
```

Comment:
Why doesn't our linter complain about this? :grief:
```suggestion
        if (BuildConfig.DEBUG) {
```

Reply:
Java...